json-rpc/src/tests/unit_tests.rs

```diff
@@ -306,15 +306,20 @@ proptest! {
         let client = reqwest::blocking::Client::new();
         let url = format!("http://{}", address);
 
-        let event_key = hex::encode(mock_db.events.iter().next().unwrap().1.key().as_bytes());
+
+        let event_index = 0;
+        let mock_db_events = mock_db.events;
+        let (first_event_version, first_event) = mock_db_events[event_index].clone();
+        let event_key = hex::encode(first_event.key().as_bytes());
         let request = serde_json::json!({"jsonrpc": "2.0", "method": "get_events", "params": [event_key, 0, 10], "id": 1});
-        let _resp = client.post(&url).json(&request).send().unwrap();
-        // TODO: flaky tests
```

Comment:
@thefallentree: I also saw flakiness here and fixed it using the scheme to the right. Adding you to make sure it looks ok.